core/src/main/java/io/smallrye/openapi/runtime/scanner/spi/AbstractParameterProcessor.java

```diff
@@ -674,10 +677,12 @@ protected void setSchemaProperties(Schema schema,
             setDefaultValue(paramSchema, getDefaultValue(paramTarget));
             TypeUtil.mapDeprecated(paramTarget, paramSchema::getDeprecated, paramSchema::setDeprecated);
 
-            BeanValidationScanner.applyConstraints(paramTarget,
-                    paramSchema,
-                    paramName,
-                    (target, name) -> setRequired(name, schema));
+            if (beanValidationScanner.isPresent()) {
+                beanValidationScanner.get().applyConstraints(paramTarget,
+                        paramSchema,
+                        paramName,
+                        (target, name) -> setRequired(name, schema));
+            }
```

Comment:
For consistence with the one above.
```suggestion
            beanValidationScanner.ifPresent(s -> s.applyConstraints(paramTarget,
                        paramSchema,
                        paramName,
                        (target, name) -> setRequired(name, schema)));
```

Reply:
This can't be done because `paramSchema` isn't effectively final.

I agree though that consistency would be good so I'll do the change the other way, replacing `ifPresent` with the `if` block.